### ISSUE TITLE
UI: Transactions: Move fee next to Launch icon

### ISSUE THREAD

**Issue by mdyring** [e-money/lunie-light#112]
To make Transaction list more accurate, I have added txs matching on market.owner to accurately consider transaction fees when orders are added (but not immediately filled).

This makes some TX look awkward esp. for unfilled orders. Could we try moving the TX fee up next to the Launch icon? (to the left side, at the top right). Ideally it should align with the amounts below still.

![image](https://user-images.githubusercontent.com/18033781/103550125-b7335900-4ea8-11eb-8f86-83859045a2c7.png)

I tried experimenting a bit myself but is not a CSS wizard. Also it seems we need to move Launch icon from MessageItem.vue -> TransactionItem.vue for it to work.

I think it is fine to leave the Memo at the bottom, in case it is set.

**Comment by Bitcoinera** [comment]
:thinking: Wouldn't it be better to include some kind of extra info like "Unfilled" ?

Otherwise, I understand what you'd like. Simply that fee appears where amounts, if there is no given amount. I will look into it

**Comment by Bitcoinera** [comment]
??

Strangely the transaction that did make effectively a swap has no fee to be seen

![image](https://user-images.githubusercontent.com/40721795/105355434-72225d00-5bf2-11eb-8147-6251787869b1.png)

:thinking:  Weird

**Comment by mdyring** [comment]
Yeah, EDIT: is correct. Trade was initiated by other user (who paid the fee) 👍
